modules/flowable-inmemory-data/src/main/java/org/flowable/engine/data/inmemory/AbstractEntityComparator.java

```diff
@@ -0,0 +1,122 @@
+/* 
+ * Copyright 2022 Qvantel Finland Oy
+ *
```

Comment:
Please see https://github.com/flowable/flowable-engine/blob/main/ide-settings/idea/fileTemplates/includes/Copyright.java for the project's standard copyright block; thanks

Reply:
The intention with Apache 2.0 license is to include the copyright owner(s) (See https://www.apache.org/licenses/LICENSE-2.0#apply) ; Copyright does not mean the licensing terms would be different - an indeed Flowable seems to contain license headers with Copyright information in place (eg. 

The copyright was not added to any _modified_ source, only to the new contributions within this PR.

Is there a legal reason to not include the Copyright part of the Apache License? Are there alternative suggestions on how to handle Copyright information? Does the Flowable Project provide some further documentation on this matter?

Reply:
Our company policy, like many others, is to include copyright information in Opensource Contributions. It does not necessarily need to be in the _copyright headers_, but that is the logical place.

Reply:
Hi @ikaakkola , thanks for the PR and the additional information about the reason of the copyright statement. For the Flowable project we only allow the standard Apache header. Instead you can mention the author above the class construct to show that you or your company has added the source code. Would that also work for you?

Reply:
Hi @ikaakkola , thanks for the follow-up feedback. What would be matching with our project standards is to use @author someone (company name) or @author company name. We don't use the word copyright in the Flowable source code and would like to keep following that pattern.

Reply:
I believe that will be fine as well. I will update the PR accordingly.